components/insight/SRC/org/openmicroscopy/shoola/env/ui/DataTransferActivity.java

```diff
@@ -109,6 +111,9 @@ protected void notifyActivityEnd()
 	{
 		if (result instanceof ProcessReport) {
 			type.setText(DESCRIPTION_ERROR);
+			GraphException ex = ((ProcessReport) result).getGraphException();
+			if (ex != null) 
+				messageLabel.setText(messageLabel.getText()+" [Details: "+ex.message+"]");
```

Comment:
maybe none at all is best, you never know what other messages might be added server-side